js/src/lib/components/Icons/IconRobotics.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	export let classNames = "";
+</script>
+
+<svg
+	class={classNames}
+	xmlns="http://www.w3.org/2000/svg"
+	xmlns:xlink="http://www.w3.org/1999/xlink"
+	aria-hidden="true"
+	fill="currentColor"
+	focusable="false"
+	role="img"
+	width="1em"
+	height="1em"
+	preserveAspectRatio="xMidYMid meet"
+	viewBox="0 0 32 32"
```

Comment:
```suggestion
	viewBox="0 0 24 24"
```

Fixes the slight top left tilt in this msg,
Now it fills the entire "square"
<img width="128" alt="image" src="https://user-images.githubusercontent.com/11827707/193679633-bd630de8-3d44-43ab-8384-90697e49b615.png">

Reply:
can we see it in context?

Reply:
I added screenshots at the top